x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/alerts_context.tsx

```diff
@@ -10,11 +10,12 @@ import React, {
   memo,
   useContext,
   useRef,
-  useMemo,
   type RefObject,
   type PropsWithChildren,
 } from 'react';
 import type { AlertsTableImperativeApi } from '@kbn/response-ops-alerts-table/types';
+import { useSpaceId } from '../../../common/hooks/use_space_id';
+import { useStoredAIAlertFlyoutShowAnonymization } from '../../../onboarding/components/hooks/use_stored_state';
```

Comment:
These "util" hooks were created only for the onboarding page sub-plugin. If we want to use them here, it would be better to move them up to a shared place. 

Or, you can also replicate the same thing with the following:

```ts
const [showAnonymizedValues = false, setShowAnonymizedValues] = useLocalStorage<boolean>(`securitySolution.aiAlertFlyout.showAnonymization.${spaceId}`)
```